components/ConvertSteps/useStepLayout.js

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect, useRef } from 'react'
+import useMeasure from 'react-use-measure'
+import { ResizeObserver as Polyfill } from '@juggle/resize-observer'
```

Comment:
Applying to cover older browsers https://caniuse.com/#feat=resizeobserver

If our support on this converter doesn't extend that far then will happily remove.